docs/HOW_TO_UPGRADE.md

```diff
@@ -244,3 +244,55 @@ These are custom modules and this is what you have to keep in mind when updating
   #### 2. Decidim Admin Extended ("decidim-admin-extended"):
 
   This module adds the necessary routes, menus and views to show the Type module inside the admin area
+
+### Overrides
```

Comment:
why a whole new section? are all the previous contents correct? or we should also update them?

Reply:
Same here.